src/crystal/system/win32/file.cr

```diff
@@ -173,22 +177,22 @@ module Crystal::System::File
     end
 
     unless exists? real_path
-      raise ::File::Error.from_errno("Error resolving real path", Errno::ENOTDIR, file: path)
+      raise ::File::Error.from_errno("Error resolving real path", file: path)
```

Comment:
Shouldn't this use `from_winerror`?

Reply:
Umm yes so this is definitely incorrect, I see now. But the prior value was also incorrect, specs failed with it.

The thing is, the function could cause both errno and winerror, so either one would be wrong.
Maybe I just need to hardcode a different value?